media/js/mozorg/home.js

```diff
@@ -165,74 +165,74 @@
         $buttons.prependTo('.extra-news > .control');
 
         $('.news-buttons .btn-next').bind('click', function() {
-            gaTrack(['_trackEvent', 'Mozilla in the News Interactions', 'Next', 'News Navigation Arrows']);
+            window.dataLayer.push({
```

Comment:
This file is still in production for some locales, but soon to be removed in PR #2998. Probably worth updating still, although alternatively we could just remove the calls to `gaTrack` altogether?